compiler/plugins/target/AMD-AIE/aie/AMDAIEObjectFifoStatefulTransform.cpp

```diff
@@ -284,589 +201,460 @@ void createDMA(
   {
     OpBuilder::InsertionGuard gg(builder);
     builder.setInsertionPointToStart(dmaBlock);
-    builder.create<DMAStartOp>(
-        builder.getUnknownLoc(),
-        static_cast<xilinx::AIE::DMAChannelDir>(channelDir), channelIndex,
-        /*repeatCount*/ 0, bdBlock, &endBlock);
+    builder.create<DMAStartOp>(builder.getUnknownLoc(), channelDir,
+                               channelIndex,
+                               /*repeatCount*/ 0, bdBlock, &endBlock);
```

Comment:
nit: Likely that this can be clang-formatted.

Reply:
It didn't when I ran clang-format on this file. However, I can manually reformat it and clang-format won't change it back.